### DISCUSSION TITLE
Ability to update nested query in more concise way

### DISCUSSION THREAD

**Discussion by hyochan** [prisma/prisma#3227] [Q&A] [answered]
```
  const users = await prisma.user.findMany({
    where: { email },
    take: 1,
  });

  if (!users || users.length === 0) return null;

  return prisma.user.update({
    data: {
      memberships: {
        update: {
          data: {
            verifiedIn: new Date(),
          },
          where: {
            membershipId_userId: {
              userId: users[0].id,
              membershipId,
            },
          },
        },
      },
    },
    where: { email },
  });
```

I think this query would be nicer if we can shorten ` userId: users[0].id` <=== this part.
Would there be any possibility to get `user.id` in `prisma.user.update` when we only know user `email`?

**Comment by ryands17** [comment, marked as answer]
Hey @hyochan 👋 
As this is a composite key meaning the `userId` is required, you would need to do it in this way itself with two queries.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
